cmd/mdmctl/apply.go

```diff
@@ -42,7 +42,10 @@ func (cmd *applyCommand) setup() error {
 
 func (cmd *applyCommand) Run(args []string) error {
 	if len(args) < 1 {
-		cmd.Usage()
+		err := cmd.Usage()
```

Comment:
would prefer to see single returns on the same line as the if
` if err := (); err != nil ...`